etc/notebooks/got_scotch_demo/scotch_dashboard.ipynb

```diff
@@ -555,6 +628,21 @@
    ]
   },
   {
+   "cell_type": "markdown",
+   "metadata": {
+    "urth": {
+     "dashboard": {
+      "hidden": true
+     }
+    }
+   },
+   "source": [
+    "Now we can render the previously assigned __tmpl__ with an argument _name_ as our first scotch 'Aberfeldy'.\n",
```

Comment:
This section is a bit confusing. Probably want to have the first paragraph as markdown above the `prompt_w` cell and the second sentence as a new markdown cell directly above the `picker_w` cell.

For the first paragraph, be sure to mention that the first scotch is just the default value and that the template will re-render whenever the user picks a new scotch from the drop down.